### PR TITLE
Add all i18n strings for all components

### PR DESCRIPTION
**Pull request by jrainville**
This means that we're ready to translate 🎉 

I also cleaned up a couple of components that were no longer used or that were too big.

(sorry this will be very painful to review)

### CODE REVIEW

File: src/js/pages/Arbitration/index.jsx [hunk 1]
```diff
@@ -124,66 +146,89 @@ class Arbitration extends Component {
...
 
-    const status = getArbitrationStatus(escrow.arbitration.result);
+    const status = this.getArbitrationStatus(escrow.arbitration.result);
     const openedByBuyer = addressCompare(escrow.arbitration.openBy, escrow.buyer);
     return (
       <div className="escrow">
-        <h2>Dispute Details <span className={"arbitrationStatus " + status}>{status}</span></h2>
+        <h2>
+          <Trans i18nKey="arbitration.disputeDetails" values={{status}}>
+            Dispute Details <span className={"arbitrationStatus " + status}>{{status}}</span>
```
Comment (richard-ramos) [inline src/js/pages/Arbitration/index.jsx]: :heavy_multiplication_x:

File: src/js/pages/Arbitration/index.jsx [hunk 2]
```diff
@@ -124,66 +146,89 @@ class Arbitration extends Component {
...
               <Col xs={3} />
             </Row>
 
             {this.renderModal()}
 
-            <ConfirmDialog display={this.state.displayDialog} onConfirm={this.resolveDispute} onCancel={this.displayDialog(false)} title="Resolve dispute" content="Are you sure?" cancelText="No" />
+            <ConfirmDialog display={this.state.displayDialog} onConfirm={this.resolveDispute}
+                           onCancel={this.displayDialog(false)} title={t('arbitration.resolve')}
+                           content={t('arbitration.youSure')}
+                           cancelText="No"/>
```
Comment (richard-ramos) [inline src/js/pages/Arbitration/index.jsx]: :heavy_multiplication_x:

File: src/js/pages/Arbitrators/index.jsx [hunk 3]
```diff
@@ -50,21 +51,25 @@ class Arbitrators extends Component {
...
         <ListGroup>
-          {Object.keys(arbitrators).length === 0 && loading && <p>Loading arbitrators...</p>}
+          {Object.keys(arbitrators).length === 0 && loading && <p>{t('arbitrators.loading')}</p>}
           {Object.keys(arbitrators).length === 0 && !loading && <Fragment>
-            <p className="mb-0">No arbitrators in the system</p>
-            <p>Become one <Link to="/arbitrator/license">here</Link></p>
+            <p className="mb-0">{t('arbitrators.noArbis')}</p>
+            <p>
+              <Trans i18nKey="arbitrators.becomeArbi">
+                Become one <Link to="/arbitrator/license">here</Link>
```
Comment (richard-ramos) [inline src/js/pages/Arbitrators/index.jsx]: :heavy_multiplication_x:

File: src/js/pages/Arbitration/index.jsx (hunk 1 above)
Comment (jrainville) [reply src/js/pages/Arbitration/index.jsx]: This is a special case where the `<Trans>` component does the magic. This let's us use tags inside the translations.

File: src/js/pages/Arbitrators/index.jsx (hunk 3 above)
Comment (jrainville) [reply src/js/pages/Arbitrators/index.jsx]: Same here for `<Trans>`

**Pull request merged**